### PR TITLE
Update aws eventbridge template to use role

### PR DESCRIPTION
**Pull request by GMadorell**
https://coralogix.atlassian.net/browse/VTX-3815

### CODE REVIEW

File: coralogix-eventbridge-policy/template.yaml [hunk 1]
```diff
@@ -6,51 +6,67 @@ Parameters:
...
+    Description: The Alias for the Coralogix region, possible options are [us1, us2, eu1, eu2, ap1, ap2, custom]
     AllowedValues:
-      - US1
-      - US2
-      - EU1
-      - EU2
-      - AP1
-      - AP2
-      - Custom
+      - us1
```
Comment (mstakic) [inline coralogix-eventbridge-policy/template.yaml:13]: This value is provided from the web-app in uppercase. Did you verify if it's correctly applied?

Comment (GMadorell) [reply coralogix-eventbridge-policy/template.yaml:13]: Yeah, I have the changes in the webapp ready too :)

Comment (GMadorell) [reply coralogix-eventbridge-policy/template.yaml:13]: https://github.com/coralogix/cx-web-workspace/pull/766

Comment (GMadorell) [reply coralogix-eventbridge-policy/template.yaml:13]: I was also trying to lowercase inside cloudformation but that was hell

**Review by mstakic — APPROVED** [review]

**Review by xBATx — APPROVED** [review]
LGTM 🎉

**Pull request merged**